tests/test_covar2d.py

```diff
@@ -23,46 +18,41 @@ def setUp(self):
 
         L = 8
         n = 32
-        C = 1
-        SNR = 1
-        pixel_size = 5
+        pixel_size = 5.0*65/L
         voltage = 200
         defocus_min = 1.5e4
         defocus_max = 2.5e4
         defocus_ct = 7
-        Cs = 2.0
-        alpha = 0.1
+
+        self.noise_var = 1.3957e-4
+        noise_filter = ScalarFilter(dim=2, value=self.noise_var)
 
         filters = [RadialCTFFilter(pixel_size, voltage, defocus=d, Cs=2.0, alpha=0.1) for d in
                    np.linspace(defocus_min, defocus_max, defocus_ct)]
 
-        # Since FFBBasis2D doesn't yet implement dtype, we'll set this to double to match its built in types.
+        vols = Volume(np.load(os.path.join(DATA_DIR, 'clean70SRibosome_vol.npy')))
```

Comment:
In the new version, we don't need to do transpose for the input map and I removed. I put back in case we want to take a look again.

Reply:
Why don't we need to transpose the input anymore?

Reply:
We need transpose previously because Garrett want to use the same binary data for comparison. But due to the different way to generate simulation images (not vol2imag) and noise, I have to update the binary data and the transpose can be removed.

Reply:
Ok. There's a certain chance those transposes would go away after the whole conversion is done, though. Right now, let's replace the binary data as you've done, but keep the `#RCOPT` comment in case something ends up breaking still.

Reply:
# RCOPT was added back in a recent commit.